didcomm_resolver/protocol/v0_9.py

```diff
@@ -118,13 +122,18 @@ async def handle(self, context: RequestContext, responder: BaseResponder):
                 f"Could not resolve DID {context.message.did} using service"
                 f" {resolver_url}"
             )
-            raise HandlerException(msg)
+            reply_msg = ResolveDIDProblemReport()
```

Comment:
ProblemReports have an `explain_ltxt` attribute with error info; we should be setting that even if we don't necessarily use it on the other end in the `map_exception` method.